web/src/beta/lib/core/Crust/Plugins/plugin_types.ts

```diff
@@ -23,6 +23,7 @@ import type {
 } from "@reearth/beta/lib/core/Map";
 
 import { Block } from "../Infobox";
+import { InteractionModeType } from "../types";
```

Comment:
We have a lot of types being imported in to this file from @reearth/beta/lib/core/Map.
Does it make sense to have this type being imported from there as well? 
Or, does it make sense to move all of the above types in to this types file to consolidate types?

Reply:
I thought the mode system should be provided by `Crust`, so I defined it in here. `Map` is for abstraction for map engine, so I think it's better to place it in `Crust`.